deploy/fly/prod.toml

```diff
@@ -40,3 +39,15 @@ processes = []
     interval = "15s"
     restart_limit = 6
     timeout = "2s"
+  
+  [[services.http_checks]]
+    interval = 10000
+    grace_period = "5s"
+    method = "get"
+    path = "/"
+    protocol = "http"
```

Comment:
@chasers we should set this to `https` if we're going to set `tls_skip_verify` to `false`

Reply:
"When true (and using HTTPS protocol) skip verifying the certificates sent by the server."

Seems to me that this should be `true` when using `https`.

Reply:
And I'm thinking these checks happen internally so unless we actually serve a cert from the instance we need to do http.  Although not sure if they're terminating https at the load balancer or what.

Reply:
Gonna give it a go and see what happens.